pkg/handler/project/project.go

```diff
@@ -612,11 +612,18 @@ func (h *handler) UpdateMember(c *gin.Context) {
 				JoinedDate:     time.Now(),
 			}
 
-			if err := h.store.ProjectHead.Upsert(tx.DB(), head); err != nil {
-				l.Error(err, "failed to upsert project head")
-				c.JSON(http.StatusInternalServerError, view.CreateResponse[any](nil, nil, done(err), head, ""))
-				return
+			_, err := h.store.ProjectHead.One(tx.DB(), projectID, body.EmployeeID.String(), model.HeadPositionTechnicalLead.String())
+			if err != nil {
+				if errors.Is(err, gorm.ErrRecordNotFound) {
+					err = h.store.ProjectHead.Create(tx.DB(), head)
+					if err != nil {
+						l.Error(err, "failed to create project head")
+						c.JSON(http.StatusInternalServerError, view.CreateResponse[any](nil, nil, done(err), head, ""))
+						return
+					}
+				}
```

Comment:
the logic here as same as update project_head, need update old project_head or create new project_head